options.html

```diff
@@ -136,6 +136,37 @@
 {
   box-shadow: 0px 0px 10px #e03030;
 }
+#userFilterLists {
+    margin-bottom: 8px;
+}
+
+#whitelistForm #newWhitelistDomain,
```

Comment:
Wouldn't you agree that it's simpler and cleaner to add a class "wide" to those elements than having to list each of them individually? That way styles could also easily be reused later on.

Reply:
+1